presto-docs/src/main/sphinx/language/types.rst

```diff
@@ -99,6 +99,16 @@ String
 
     Example type definitions: ``varchar``, ``varchar(20)``
 
+    SQL statements support string value usage as simple literals ``Hello
+    world``. To use Unicode literals within a string, you have to use the prefix
```

Comment:
Grammar nit: use "must use" rather than "have to use" as it expresses a specific obligation

However, this is incorrect, because it's legal to put arbitrary Unicode characters inside a string literal. The Unicode escape syntax is an alternate way of writing the string. My original suggestion was correct in this regard.

Reply:
I am not sure what you are saying now. What original suggestion? The example I had with the U& inside the literal normal string? Are both ways supposed to work now? If you have time .. ping me on slack for a call